### PR TITLE
Updated fromPubkey to use PublicKey in the transfer instruction

### PR DESCRIPTION
**Pull request by husna3249**
### Changes Made
- Changed `fromPubkey: account` to `fromPubkey: new PublicKey(account)` to ensure proper handling of the public key.

### Reason for Change
Using `new PublicKey(account)` ensures that the account variable is properly converted to a PublicKey instance, preventing potential issues with the transfer instruction.

### CODE REVIEW

File: content/courses/solana-pay/solana-pay.md [hunk 1]
```diff
@@ -235,11 +235,15 @@ async function post(req: PublicKey, res: PublicKey) {
     feePayer: account,
   });
 
-  const instruction = SystemProgram.transfer({
+    const instruction = SystemProgram.transfer({
     fromPubkey: account,
+    fromPubkey: new PublicKey(account),
```
Comment (mikemaccana) [inline content/courses/solana-pay/solana-pay.md:240]: You're adding a second `fromPubkey`. Also the indentation is broken.

**Review by mikemaccana — CHANGES_REQUESTED** [review]
See comments

File: content/courses/solana-pay/solana-pay.md (hunk 1 above)
Comment (husna3249) [reply content/courses/solana-pay/solana-pay.md]: Thank you for your feedback! I've updated the code to remove the duplicate and fixed the indentation.

File: content/courses/solana-pay/solana-pay.md [hunk 2]
```diff
@@ -235,11 +235,14 @@ async function post(req: PublicKey, res: PublicKey) {
     feePayer: account,
   });
 
-  const instruction = SystemProgram.transfer({
-    fromPubkey: account,
+    const instruction = SystemProgram.transfer({
+    fromPubkey: new PublicKey(account),
```
Comment (mikemaccana) [inline content/courses/solana-pay/solana-pay.md:239]:
```suggestion
  const instruction = SystemProgram.transfer({
    fromPubkey: new PublicKey(account),
```

File: content/courses/solana-pay/solana-pay.md [hunk 3]
```diff
@@ -235,11 +235,14 @@ async function post(req: PublicKey, res: PublicKey) {
...
     toPubkey: Keypair.generate().publicKey,
     lamports: 0.001 * LAMPORTS_PER_SOL,
-  });
+    });
+
+
+
```
Comment (mikemaccana) [inline content/courses/solana-pay/solana-pay.md:245]:
```suggestion
  }); 
```

**Review by mikemaccana — APPROVED** [review]

**Comment by mikemaccana** [comment]
Thanks! You still missed the indentation but I did it for you. Merged!

**Pull request merged**